app/views/appointments/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1>Overdue patients</h1>
+<h1>Overdue Patients</h1>
```

Comment:
We use sentence case throughout this app; please don't change this.

Reply:
will fix this